ipwb/replay.py

```diff
@@ -62,6 +74,60 @@ def setServerHeader(response):
     return response
 
 
+def allowed_file(filename):
+    return filename.lower().endswith(ALLOWED_EXTENSIONS)
+
+
+@app.route('/upload', methods=['POST'])
+def upload_file():
+    # check if the post request has the file part
+    if 'file' not in request.files:
+        flash('No file part')
+        return redirect(request.url)
+    file = request.files['file']
+    # if user does not select file, browser also
+    # submit an empty part without filename
+    if file.filename == '':
+        flash('No selected file')
+        return redirect(request.url)
+    if file and allowed_file(file.filename):
+        filename = secure_filename(file.filename)
+        warcPath = os.path.join(app.config['UPLOAD_FOLDER'], filename)
+        file.save(warcPath)
+
+        random.seed()
+
+        cdxjPath = '/tmp/' + ''.join(random.sample(
+            string.ascii_uppercase + string.digits * 6, 12)) + '.cdxj'
+        combinedcdxjPath = '/tmp/' + ''.join(random.sample(
+            string.ascii_uppercase + string.digits * 6, 12)) + '.cdxj'
```

Comment:
Rm both `cdxjPath` and `combinedcdxjPath` and perhaps `random.seed()` will not be needed here either.

Continue...